sdp-utils.c

```diff
@@ -641,6 +641,38 @@ int janus_sdp_get_codec_pt(janus_sdp *sdp, const char *codec) {
 	return -1;
 }
 
+gboolean janus_sdp_attribute_fec_enable(janus_sdp *sdp, const int codec_pt) {
```

Comment:
See discussion made above: this whole new method is not needed, we're already configuring FEC when creating the attribute using the SDP utils:

	janus_sdp_attribute *a = janus_sdp_attribute_create("fmtp",
		"%d maxplaybackrate=%"SCNu32"; stereo=0; sprop-stereo=0; useinbandfec=%d\r\n",
			participant->opus_pt, participant->room->sampling_rate, participant->fec ? 1 : 0);

Reply:
Ok